packages/trader/src/App/Components/Elements/PositionsDrawer/__tests__/position-modal-card.spec.tsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { mockStore } from '@deriv/stores';
+import { TCoreStores } from '@deriv/stores/types';
+import { VANILLALONG, TURBOS } from '@deriv/shared';
+import { useTraderStore } from 'Stores/useTraderStores';
+import PositionsModalCard from '../positions-modal-card';
+import TraderProviders from '../../../../../trader-providers';
+
+const default_mock_props = {
+    className: 'test_className',
+    contract_info: {
+        contract_type: VANILLALONG.CALL,
+        underlying: '',
+        contract_id: 123386875,
+        currency: 'USD',
+        buy_price: 2671,
+        bid_price: 2517,
+        entry_spot: 2666,
+        barrier: '2650',
+        is_sold: 0,
+        date_start: 123532989,
+        date_expiry: 626512765,
+        tick_count: 15,
+    },
+    contract_update: 'test_contract_update',
+    currency: 'USD',
+    current_tick: 6,
+    id: 123386875,
+    is_loading: false,
+    is_sell_requested: false,
+    is_unsupported: true,
+    onClickSell: jest.fn(),
+    profit_loss: 35.6786,
+    onClickCancel: jest.fn(),
+    togglePositions: jest.fn(),
+    toggleUnsupportedContractModal: jest.fn(),
+} as unknown as React.ComponentProps<typeof PositionsModalCard>;
+
+const default_mock_store = {
+    modules: {
+        trade: {
+            active_symbols: ['Vol 10'] as unknown as ReturnType<typeof useTraderStore>['active_symbols'],
```

Comment:
```suggestion
            active_symbols: [{ symbol: 'R_10' }] as ActiveSymbols,
```
and import { ActiveSymbols } from '@deriv/api-types';

Reply:
Will do, thank you